### PR TITLE
Feature/client notes table Co-authored-by: wild-kard

### PR DESCRIPTION
**Pull request by stkirk**
https://www.loom.com/share/4c7df1dced8741418ae70a9079301426

- A migration for a case notes table called client_notes was added and the new table seeded with dummy data. The profiles table was seeded as well so that notes can correspond to case worker profile. Case notes can now be stored in the database

- What feature / user story is it for?
- https://trello.com/c/3UKrfP4K/21-as-a-case-manager-i-want-to-be-able-to-update-a-case-with-notes-so-that-i-can-remain-up-to-date-on-the-case

- [x] New feature (non-breaking change which adds functionality)

## Change Status

- [ ] Complete, tested, ready to review and merge
- [x] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [ ] Yes
- [ ] No
- [x] Not necessary

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: data/seeds/04-client_notes.js [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+        {
+          client_id: 5,
+          source_view: 'caseview',
+          message:
+            'Proin sollicitudin varius ultrices hac sem Pharetra nibh leo dapibus ligula placerat egestas consequat mi. Curae; curae; suscipit convallis tellus dictum aliquam diam orci ad. Nam risus eu quis massa, a praesent gravida nam iaculis hendrerit malesuada bibendum id phasellus.',
+          created_by: '8',
+        },
+      ]);
+    });
+};
```
Comment (chavionjackson) [inline data/seeds/04-client_notes.js:340]: Everything looks good!

File: data/migrations/20220218164416_create_client_notes.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      table.text('message').notNullable();
+      table.timestamp('created_at').defaultTo(knex.fn.now());
+      table.timestamp('deleted_at').defaultTo(null);
+      table.string('created_by');
+    });
+};
+
+exports.down = function (knex) {
+  return knex.schema.dropTableIfExists('client_notes');
+};
```
Comment (chavionjackson) [inline data/migrations/20220218164416_create_client_notes.js:22]: Nice  migrations

**Review by chavionjackson — APPROVED** [review]

File: data/migrations/20220218164416_create_client_notes.js (hunk 2 above)
Comment (thecuriousones) [reply data/migrations/20220218164416_create_client_notes.js:22]: Migrations looks good and follows the schema

File: data/seeds/04-client_notes.js (hunk 1 above)
Comment (thecuriousones) [reply data/seeds/04-client_notes.js:340]: Both seed files are looking great and ready to use!

**Review by thecuriousones — APPROVED** [review]

**Review by ashtilawat — APPROVED** [review]

**Pull request merged**